src/oscar/apps/offer/abstract_models.py

```diff
@@ -625,8 +627,16 @@ def round(self, amount):
         """
         Apply rounding to discount amount
         """
-        if hasattr(settings, 'OSCAR_OFFER_ROUNDING_FUNCTION'):
-            return settings.OSCAR_OFFER_ROUNDING_FUNCTION(amount)
+        rounding_function = getattr(settings, 'OSCAR_OFFER_ROUNDING_FUNCTION', None)
+        if callable(rounding_function):
+            return rounding_function(amount)
```

Comment:
Please add a deprecation warning here:

    warnings.warn("Defining OSCAR_OFFER_ROUNDING_FUNCTION as a function is deprecated. Specify a dotted path to the function instead." , RemovedInOscar21Warning, stacklevel=1)

Please also update the [documentation](https://github.com/django-oscar/django-oscar/blob/f2bb0fb73c83cb879fb2dec9184059dd96d814e0/docs/source/ref/settings.rst#offer-settings) to say that this expects a dotted path.